### PR TITLE
docs(bootstrap): change link to Movie sample data

### PR DESCRIPTION
**Pull request by wafuwafu13**
*Issue #, if available:*

*Description of changes:*

`https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.*.html` has moved to https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.html.

Movie sample data is not used in https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.html.

```zsh
$ curl https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.NodeJs.02.html
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>302 Found</title>
</head><body>
<h1>Found</h1>
<p>The document has moved <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.html">here</a>.</p>
</body></html>

$ curl https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.Ruby.02.html
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>302 Found</title>
</head><body>
<h1>Found</h1>
<p>The document has moved <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.html">here</a>.</p>
</body></html>

$ curl https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.Python.02.html
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>302 Found</title>
</head><body>
<h1>Found</h1>
<p>The document has moved <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.html">here</a>.</p>
</body></html>
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by StoneDot** [comment]
It appears that `movies.json` has been relocated to [GitHub](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/resources/sample_files). I would recommend updating the links for the users' reference. What do you think about this suggestion?

**Comment by wafuwafu13** [comment]
That's make sense https://github.com/awslabs/dynein/pull/184/commits/73eacb78ace635b6d2e8fb88cb17570cb4a7337b

File: README.md [hunk 1]
```diff
@@ -431,7 +431,7 @@ using_table: ~
 
 ## Working with DynamoDB items
 
-As an example let's assume you have [official "Movie" sample data](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.Python.02.html). To prepare the table with data loaded, simply you can execute `dy bootstrap --sample movie`.
+As an example let's assume you have [official "Movie" sample data](https://raw.githubusercontent.com/awsdocs/aws-doc-sdk-examples/main/resources/sample_files/movies.json). To prepare the table with data loaded, simply you can execute `dy bootstrap --sample movie`.
```
Comment (StoneDot) [inline README.md:434]: We can use permalink as the more stable link. For example, you can use the following link.
https://raw.githubusercontent.com/awsdocs/aws-doc-sdk-examples/c2edcff1365d4b454b51075d632a1be844dd3e47/resources/sample_files/movies.json

File: src/bootstrap.rs [hunk 2]
```diff
@@ -138,9 +138,8 @@ Private functions
 async fn launch_movie_sample(cx: app::Context) -> Result<(), DyneinBootstrapError> {
     println!(
         "\
-Bootstrapping - dynein will creates 'Movie' table used in public tutorials:
-e.g. https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.NodeJs.02.html
-     https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.Ruby.02.html
+Bootstrapping - dynein will creates 'Movie' table with official 'Movie' sample data:
+https://raw.githubusercontent.com/awsdocs/aws-doc-sdk-examples/main/resources/sample_files/movies.json
```
Comment (StoneDot) [inline src/bootstrap.rs:142]: We can change this URL into the permalink. But, this is just a comment, so current URL is maybe preferable due to easy-to-understand. Please select your preferable choice.

**Review by StoneDot — COMMENTED** [review]
Thank you for raising the improvement of docs. Could you check my comments?

File: src/bootstrap.rs (hunk 2 above)
Comment (wafuwafu13) [reply src/bootstrap.rs]: change to dynein README.md link

File: src/bootstrap.rs [hunk 3]
```diff
@@ -138,13 +138,13 @@ Private functions
 async fn launch_movie_sample(cx: app::Context) -> Result<(), DyneinBootstrapError> {
     println!(
         "\
-Bootstrapping - dynein will creates 'Movie' table used in public tutorials:
-e.g. https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.NodeJs.02.html
-     https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.Ruby.02.html
+Bootstrapping - dynein will creates 'Movie' table with official 'Movie' sample data:
```
Comment (StoneDot) [inline src/bootstrap.rs:141]: nits: creates → create

**Review by StoneDot — DISMISSED** [review]
LGTM with small nits (typo).

**Review by StoneDot — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**